hmf-common/src/main/java/com/hartwig/hmftools/common/linx/LinxDataLoader.java

```diff
@@ -121,7 +125,9 @@ private static LinxData load(@NotNull String somaticStructuralVariantTsv, @NotNu
                 .reportableSomaticBreakends(reportableSomaticBreakends)
                 .somaticHomozygousDisruptions(somaticHomozygousDisruptions)
                 .fusionClusterIds(fusionClusterIds)
-                .putAllSvIdToClusterId(svIdToClusterId)
+                .putAllSvIdToClusterId(visSvData.svIdToClusterId)
+                .putAllClusterIdToChainCount(visSvData.clusterIdToChainCount)
```

Comment:
Minor and up for debate, but I'd use getters for these (and make the properties private in VisSvData)

Another minor, maybe just set the properties in the builder instead of using `putAll`? To me, `putAll` gives the impression that is more is coming (you are putting entries into a growing map)